tests/unit/dags/test_create_project_from_dataset_version_dag.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+import pytest
+from airflow.models import DagBag
+
+from datarobot_provider.example_dags.datarobot_create_project_from_dataset_version_dag import (
+    create_project_from_dataset_version,
+)
+
+
+@pytest.fixture()
+def dagbag(provider_dir):
+    return DagBag(dag_folder=f"{str(provider_dir)}/example_dags", include_examples=False)
+
```

Comment:
This is common in all the unittests right? You can create a `conftest.py` in this directory and move this code here and then the fixture will be reusable across all other files in that directory.